exchanges/execute/src/execute.ts

```diff
@@ -0,0 +1,60 @@
+import { pipe, share, filter, map, fromPromise, mergeMap, merge } from 'wonka';
+import {
+  GraphQLSchema,
+  GraphQLFieldResolver,
+  GraphQLTypeResolver,
+  execute,
+} from 'graphql';
+import { Exchange, makeResult, makeErrorResult } from '@urql/core';
+
+interface ExecuteExchangeArgs {
+  schema: GraphQLSchema;
+  rootValue?: any;
+  contextValue?: any;
```

Comment:
It’d be great to just call this just `context` and make it accept a function returning an object as well, inline with Apollo Server and other servers 🙌

Same for `rootValue`; it’d be great to have that aligned with Apollo Server as well

Reply:
The naming is identical to the [graphqljs execute](https://github.com/graphql/graphql-js/blob/master/src/execution/execute.js#L146) function.

Reply:
Accepts a function now.